docs/metasploit-framework.wiki/Creating-Your-First-PR.md

```diff
@@ -0,0 +1,122 @@
+# Creating Your First PR - An Intro To Git and the PR Process
+## Intro
+Congratulations fellow traveler, so your interested in contributing to Metasploit eh? Well welcome aboard, its going to be a fun ride!
```

Comment:
```suggestion
Congratulations fellow traveler, so you're interested in contributing to Metasploit eh? Well welcome aboard, its going to be a fun ride!
```

Reply:
Resolved in https://github.com/rapid7/metasploit-framework/pull/16949/commits/2bfcac8d8630c70097cd0b029271ec9c25e522d6